### DISCUSSION TITLE
Hplip cannot be compiled in the fw after 7.13

### DISCUSSION THREAD

**Discussion by sestarus13** [Freetz-NG/freetz-ng#1070] [Allgemein]
Is there a way to integrate hplip in the firmware after 7.13 in order to support scanner function on some hp mfc low cost models only with a usb port? Is it possible to cross compile it or use some kind of emulation mips <-> x86?

thanks. :)

**Comment by fda77** [comment]
I dont support DEB shit

**Reply by sestarus13** [reply]
Not sure I understood it, but thanks for prompt reply! :)

**Comment by fda77** [comment]
Just pay aka donate to @sammy-lacht aka @MasterRoCcO to be its "friend". Then you get access to paid freetz patches

**Reply by sestarus13** [reply]
Great, how should I do this? How much should I donate?

**Comment by fda77** [comment]
Ask the seller

**Reply by sestarus13** [reply]
I just paid 5E donation to DEB, what would be the next step?

**Comment by fda77** [comment]
Read my first comment here again:

> I dont support DEB shit

https://github.com/Freetz-NG/freetz-ng/discussions/1070#discussioncomment-11200488

https://freetz-ng.github.io/freetz-ng/ADDONS.html#nervt-mit-dem-krempel-nicht-herum
